spec/classes/rabbitmq_spec.rb

```diff
@@ -287,6 +287,21 @@
             )
           end
         end
+        context 'with service_manage set to true and mqa_download_opts specified' do
+          let(:params) do
+            {
+              admin_enable: true,
+              mqa_download_opts: ['blah', 'blig']
```

Comment:
```
spec/classes/rabbitmq_spec.rb:301:33: C: Style/WordArray: Use %w or %W for an array of words. (b)
              download_options: ['blah', 'blig']
                                ^^^^^^^^^^^^^^^^
```

You just need to do `%w(blah blig)` instead. You can run rubocop locally with `bundle exec rake rubocop` if you want to make sure rubocop passes before pushing and having Travis run again.